MSAL/src/MSALResult.m

```diff
@@ -81,7 +82,7 @@ + (MSALResult *)resultWithTokenResult:(MSIDTokenResult *)tokenResult
     }
 
     MSIDAccount *resultAccount = tokenResult.account;
-    NSString *tenantId = [tokenResult.authority.url msidTenant];
+    NSString *tenantId = tokenResult.tokenResponse.idTokenObj.realm;
```

Comment:
Will token response always be present? (e.g. in case of silent token lookups)

Reply:
Done. Changed code to extract claims from rawIdToken.